files/ansible/provisioning_job.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+          - name: Provision with localhost
+            shell: "{{ workspace }}/mr-provisioner-client/mrp_client.py --mrp-url={{ mrp_url | quote }} --mrp-token={{ mrp_token | quote }} state --action provision --machine {{ item | quote }} --preseed-name {{ preseed_name | quote }} --initrd-desc {{ initrd_desc | quote }} --kernel-desc {{ kernel_desc | quote }} --kernel-opts {{ kernel_opts | quote }} --arch {{ arch | quote }} --subarch {{ subarch | quote }}"
+            with_items:
+                    - "{{ machines }}"
+
+# Has to be separated, else the above task is run once per host (meaning MrP will receive number of "hosts" provision requests)
```

Comment:
And this will spawn a job per machine, in parallel, right?

Reply:
This will spawn run one mrp_client request per machine sequentially.
The other playbook (- hosts: targets) will spawn a wait_for_connection job per machine in parallel.

Reply:
Nope, we're waiting for machines to finish in parallel.
We do have to wait for mrp_client to complete its request before doing another one.